test/run_test.py

```diff
@@ -64,13 +64,11 @@
     'function_schema',
 ]
 
-# skip < 3.3 because mock is added in 3.3 and is used in rpc_fork and rpc_spawn
+# skip < 3.3 because mock is added in 3.3 and is used in rpc_spawn
 # skip python2 for rpc and dist_autograd tests that do not support python2
 if PY33:
     TESTS.extend([
-        'rpc_fork',
         'rpc_spawn',
-        'dist_autograd_fork',
```

Comment:
Can you do the same to `test_dist_optimizer_fork.py`. It depends on `RpcAgent` as well.

Reply:
will do